src/backend/README.md

```diff
@@ -1,26 +1,108 @@
-The backend is clio's view into the database. The database could be either PostgreSQL or Cassandra.
-Multiple clio servers can share access to the same database. 
```

Comment:
Let's reset this file. If your local master branch is up to date with your remote master branch, you can run:
```
git checkout master -- src/backend/README.md
```